### DISCUSSION TITLE
I didn't pass the education plan, maybe i take the wrong photo.But when i want to try again, it doesn't work!

### DISCUSSION THREAD

**Discussion by pengrui-art** [community/community#139917] [GitHub Education] [answered]
### Hi everyone,

![image](https://github.com/user-attachments/assets/1f069eee-9e00-4df9-852b-f68c4507394e)
as you can see, it will stuck at the continue, and never can enter to the next step.

**Comment by AidenJamesYt** [comment, marked as answer]
Click "select this school"
